### PR TITLE
renamed Werbematerial to Infomaterial

### PR DESCRIPTION
**Pull request by midzer**
In my opinion "Infomaterial" sounds better as we are uncommercial.

### CODE REVIEW

File: _posts/2015-07-17-infomaterial.markdown [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+categories: freifunkmuc
+---
+
+**Infomaterial**
+
+Wir haben für Euch einige Vorlagen und Logos [hier][verbreiten] zusammengestellt. 
+
+Eine vollständige Sammlung der Vorlagen ist auf [github] zu finden.
+
+[hier]: http://freifunk-muenchen.de/verbreiten/
```
Comment (fpletz) [inline _posts/2015-07-17-infomaterial.markdown]: This should be called `[verbreiten]`.

**Comment by fpletz** [comment]
I'm not sure if we should change the URL of an already published blog post, but nobody will probably link to it anyway… :panda_face:

File: _posts/2015-07-17-infomaterial.markdown (hunk 1 above)
Comment (midzer) [inline _posts/2015-07-17-infomaterial.markdown]: true %)

**Comment by midzer** [comment]
I think this minor blog post is not that essential and hopefully not linked somewhere.
Next time i have to consider this...

**Pull request merged**